tools/scripts/ansible/Vagrantfile

```diff
@@ -2,6 +2,10 @@
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
+  config.vm.provider "virtualbox" do |v|
+    v.memory = 1024
+    # v.cpus = 2
```

Comment:
I got an out-of-memory error unless set v.memory here -- but not sure we want to change v.cpus